templates/main.template

```diff
@@ -100,165 +126,239 @@ Parameters:
     AllowedPattern: ^[0-9a-zA-Z]+([0-9a-zA-Z-.]*[0-9a-zA-Z])*$
     ConstraintDescription: Quick Start bucket name can include numbers, lowercase
       letters, uppercase letters, periods (.), and hyphens (-). It cannot start or
-      end with a hyphen (-).
+      end with a hyphen (-). If you are unsure, do not change this value.
     Default: aws-quickstart
     Description: S3 bucket name for the Quick Start assets. Quick Start bucket name
       can include numbers, lowercase letters, uppercase letters, and hyphens (-).
-      It cannot start or end with a hyphen (-).
+      It cannot start or end with a hyphen (-). If you are unsure, do not change this value.
     Type: String
   QSS3KeyPrefix:
     AllowedPattern: ^[0-9a-zA-Z-/]*$
     ConstraintDescription: Quick Start key prefix can include numbers, lowercase letters,
-      uppercase letters, hyphens (-), and forward slash (/).
+      uppercase letters, hyphens (-), and forward slash (/). If you are unsure, do not change this value.
     Default: quickstart-compliance-pci/
     Description: S3 key prefix for the Quick Start assets. Quick Start key prefix
       can include numbers, lowercase letters, uppercase letters, hyphens (-), and
-      forward slash (/).
+      forward slash (/). If you are unsure, do not change this value.
     Type: String
 Mappings:
-  CustomVariables:
-    vResourceEnvironmentTagKey:
-      Value: Environment
-    vResourceEnvironmentTagValue:
-      Value: development
   RegionServiceSupport:
```

Comment:
I'd love to chat about this a bit. I don't see this pattern as something that we can maintain easily. I'd prefer this QS only launch in regions where everything is supported. We have mechanisms for auditing/reporting from this angle.

Reply:
The only region not supported here for glacier is sa-east-1, but that is not really relevant for this template. I agree that we should only deploy where supported.